### DISCUSSION TITLE
frontend setup using shell scripting topics

### DISCUSSION THREAD

**Discussion by Bhashitha19** [learndevopsonline/learndevopsonline#1847] [Q&A]

**Comment by Bhashitha19** [comment]
![Screenshot (35)](https://github.com/learndevopsonline/learndevopsonline/assets/148459705/0edf7a9c-160f-4d62-bd93-92983989f9f6)

**Comment by chaithanya1812** [comment]
Hello Bhashitha,

can you explain, what is the error you got?

**Comment by Bhashitha19** [comment]
sir while iam doing the frontend setup using shell scripting topics i use
one of the shell scripting topics i.e.,EXIT
Exit command is returning 1 for expense.config file for remaining all it
returns 0. so, Whats the error raised i cant know that
Could you please help me sir

**Comment by chaithanya1812** [comment]
Can you join lab session once

Issue:- Using Capital letters

![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/86c8ef13-f75f-4d10-ba30-31f42ff77c63)

**Comment by Bhashitha19** [comment]
Yes, sure. but how to join lab session sir is there any link to join

**Reply by chaithanya1812** [reply]
# Lab session link
https://learndevopsonline.my.canva.site/lab-help
